javascript/javascript1/week4/voiceassistant.js

```diff
@@ -0,0 +1,72 @@
+const Name = [];
+const toDoList = [];
+
+function getReply(command) {
+  for (i = 0; i < Name.length; i++);
+
+  const NewName = "benjamin";
+  Name.push(NewName);
+
+  if (command.includes("benjamin")) {
+    console.log(`hi my name is ${NewName}`);
+    return `hello  ${NewName} nice to meet you`;
+  } else if (command === "what is my name?") {
```

Comment:
When you get `what is my name?`command , you should check if the name has been presented before, if yes , then return the name otherwise say that you should present yourself.